Source/gui/SNTBinaryMessageWindowView.swift

```diff
@@ -383,7 +383,7 @@ struct SNTBinaryMessageWindowView: View {
       )
     } else if !filePath.isEmpty {
       msg = NSLocalizedString(
-        "authorize execution of " + filePath,
+        "authorize execution of " + (filePath as NSString).lastPathComponent,
```

Comment:
I see your point but the full path is in the "More Details" section of the original dialog. And I think the majority of users aren't going to actually look at the path in this prompt because it's too long:

![image](https://github.com/user-attachments/assets/b18391c6-749f-45c7-83e8-e5dfa7dc7d4f)

Reply:
I don't disagree. The right answer here is probably doc updates to emphasize being careful.